test/extensions/filters/http/aws_request_signing/config_test.cc

```diff
@@ -21,6 +21,10 @@ TEST(AwsRequestSigningFilterConfigTest, SimpleConfig) {
   const std::string yaml = R"EOF(
 service_name: s3
 region: us-west-2
+match_excluded_headers:
```

Comment:
Can you actually test this populates the config correctly?

Reply:
done!

Reply:
done